build.rs

```diff
@@ -5,6 +5,11 @@ use std::process::Command;
 fn main() {
     #[cfg(feature = "llvm")]
     {
+        Command::new("make")
```

Comment:
We now require `make` for building solang, we could mention this in the docs.

While on any developer setup it should be already be installed anyways. But it would be nice to mention for example if someone trying to package it or use it in a container.

Reply:
I have been wondering whether this could be replaced with all the commands for building the stdlib, and doing away with the `Makefile`, and along with that the `make` dependency. I was kind of undecided on that.